index.rst

```diff
@@ -304,8 +304,8 @@ Full Table of Contents
    documenting
    silencewarnings
    fixingissues
-   tracker
-   triaging
+   tracker2
```

Comment:
Just curious of the renaming from `tracker` to `tracker2`. Is this because we're keeping a backup of the old `tracker` document? Does `2` means like `version 2` or something?

Reply:
I think @ambv did that so it's easier to review the new document as a whole, because it's nearly a full rewrite. Unfortunately, GitHub in the review pane shows a diff between tracker and tracker2 anyway.

Reply:
Yes, @JelleZijlstra is right. Git itself is trying to be too smart here but fortunately I foresaw this and the deletion is a separate commit. So if you only look at [699f8f1](https://github.com/python/devguide/pull/814/commits/699f8f1d586e88c64c23268c4bfada962342500b), it reads easier.